test/unit/models/user.spec.ts

```diff
@@ -11,12 +17,18 @@ describe("Models.User", () => {
   let clock: sinon.SinonFakeTimers;
   let today: Date;
   let user: IUser;
+  let db: Model<IUserModel>;
 
   beforeEach(() => {
+    container.unbind(TYPES.MongoConnector);
```

Comment:
As the container is instantiated with the app configuration, there is all ready a binding for the `TYPES.MongoConnector`. Because of this, we need to unbind it before running the test.